runtime/pom.xml

```diff
@@ -400,6 +400,25 @@
       <artifactId>okhttp-eventsource</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>org.projectlombok</groupId>
+        <artifactId>lombok</artifactId>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
```

Comment:
There was no discussion but it was the path of least resistance to get moving IMO. There's not much to our data layer right now so easy to change later, as long as we keep it separated and expose through interfaces.